### PR TITLE
[2.3] [Contributing] Added note about empty returns

### PR DESCRIPTION
**Pull request by phansys**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | no |
| Applies to |  |
| Fixed tickets |  |

### CODE REVIEW

**Comment by WouterJ** [comment]
:+1:

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -112,6 +112,9 @@ Structure
 * Add a blank line before ``return`` statements, unless the return is alone
   inside a statement-group (like an ``if`` statement);
 
+* Use just ``return;`` instead ``return null;`` when a function must return void
```
Comment (xabbuh) [inline contributing/code/standards.rst]: instead of

Comment (phansys) [inline contributing/code/standards.rst]: Thanks @xabbuh.

**Comment by xabbuh** [comment]
Thank you Javier.

**Comment by phansys** [comment]
Thank you too Christian.

**Pull request merged**